tests/paramgrill.c

```diff
@@ -972,7 +985,25 @@ int main(int argc, const char** argv)
         if(!strcmp(argument,"--no-seed")) { g_noSeed = 1; continue; }
 
         /* Decode command (note : aggregated commands are allowed) */
-        if (argument[0]=='-') {
+        if (longCommandWArg(&argument, "--zstd=")) {
+            g_singleRun = 1;
+            g_params = ZSTD_getCParams(2, g_blockSize, 0);
```

Comment:
This will initialize `g_params` using compression parameters from level 2.
Thus, if the user only provide a few arguments (for example only `wlog`), all other parameters will be initialized with values from level 2.

The problem is, that's too limited.

With the older scheme, it was possible to start from a different compression level, 
for example `SL9` meaning "use parameters from level 9", 
and then add a few modified parameters,
for example `SL9w18` meaning "start from level 9, and just modify `windowLog=18`".

We don't have this ability here, which is very limiting.

Suggestion : add a new sub-command `level=` to achieve same capability.